WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -1010,14 +1015,18 @@ decisions = {
 			independent = yes
 			OR = {
 				has_landed_title = k_persia
+				any_realm_lord = { has_landed_title = k_persia }
 				has_landed_title = k_khiva
+				any_realm_lord = { has_landed_title = k_khiva }
 				has_landed_title = k_afghanistan
+				any_realm_lord = { has_landed_title = k_afghanistan }
 				has_landed_title = k_baluchistan
+				any_realm_lord = { has_landed_title k_baluchistan }
```

Comment:
No, the vassal would do it, as happened historically. Otherwise one can build directly. It is meant to be a more difficult religion.